PassKitUI/PKPGSVSectionHeaderView.h

```diff
@@ -0,0 +1,8 @@
+#import <Foundation/Foundation.h>
```

Comment:
UIKit imports Foundation so I'm not sure this is necessary

Reply:
UIKit didnt seem to be able to be imported here but yea you are right